tool/analyze-and-test-examples.sh

```diff
@@ -116,7 +116,7 @@ function analyze_and_test() {
   if [[ -z $TEST_FILES ]]; then
     echo "No browser-only tests."
   else
-    travis_fold start analyzeAndTest.tests.browser
+    echo "::group::analyzeAndTest.tests.browser"
     echo Running browser tests ...
     PLATFORM=chrome
     if [[ -n $TRAVIS ]]; then PLATFORM=travischrome; fi
```

Comment:
Still a travis reference?

Reply:
Thanks, not sure how I missed that one. Though, the browser tests pass on GitHub Actions without that `--no-sandbox` so I just removed the whole travischrome noise.